### PR TITLE
rpc: getvalidators -> getnextblockvalidators

### PR DESCRIPTION
**Pull request by AnnaShaleva**
We don't have `getvalidators` RPS call, there's only
`getnextblockvalidators` in the reference implementation.

### CODE REVIEW

**Comment by roman-khimov** [comment]
That's from neo-project/neo-modules#338, I was not expecting it to change anything like that. But there is a bug in the C# client then, it still has `GetValidators` (I'll submit that).

File: pkg/rpc/client/client.go [hunk 1]
```diff
@@ -57,8 +57,8 @@ type cache struct {
 // calculateValidUntilBlockCache stores cached number of validators and
 // cache expiration value in blocks
 type calculateValidUntilBlockCache struct {
-	validatorsCount uint32
-	expiresAt       uint32
+	committeeCount uint32
```
Comment (roman-khimov) [inline pkg/rpc/client/client.go:60]: But why this change? We still can get the number of validators from `getnextblockvalidators`

File: pkg/rpc/server/server.go [hunk 2]
```diff
@@ -906,13 +906,13 @@ func (s *Server) getUnclaimedGas(ps request.Params) (interface{}, *response.Erro
 	}, nil
 }
 
-// getValidators returns the current NEO consensus nodes information and voting status.
-func (s *Server) getValidators(_ request.Params) (interface{}, *response.Error) {
+// getNextBlockValidators returns validators for the next block with voting status.
+func (s *Server) getNextBlockValidators(_ request.Params) (interface{}, *response.Error) {
 	var validators keys.PublicKeys
 
-	validators, err := s.chain.GetValidators()
```
Comment (roman-khimov) [inline pkg/rpc/server/server.go:913]: We need to look at other users of `GetValidators` then, maybe it's no longer relevant there either.

File: pkg/rpc/client/client.go (hunk 1 above)
Comment (AnnaShaleva) [reply pkg/rpc/client/client.go:60]: Can we guarantee that `getnextblockvalidators` never returns an empty array?

Comment (roman-khimov) [reply pkg/rpc/client/client.go:60]: The request can fail, but the network can't really work without validators, so they always exist.

File: pkg/rpc/server/server.go (hunk 2 above)
Comment (AnnaShaleva) [reply pkg/rpc/server/server.go:913]: DBFT still uses this method, we can't remove it.

Comment (roman-khimov) [reply pkg/rpc/server/server.go:913]: I don't think it needs to. It wants next block validators too.

**Review by roman-khimov — APPROVED** [review]
